gke-platform/modules/gke_autopilot/variables.tf

```diff
@@ -30,6 +30,14 @@ variable "cluster_name" {
   default     = "ml-cluster"
 }
 
+variable "cluster_labels" {
+  type        = map
+  description = "GKE cluster labels"
+  default     =  {
+    created-by = "ai-on-gke"
```

Comment:
Initially had this as:
```
created-by: github.com/GoogleCloudPlatform/ai-ongke
```

But lable values can only be lower case and special characters like `.` and `/` are not allowed.